src/irc/__init__.py

```diff
@@ -130,6 +130,7 @@ def _get(data, key, format_found=None, default=None):
     if key in data:
         value = data[key]
         if value:
+            value = _sanitize(value)
```

Comment:
Thanks this is needed!

I think this is the wrong spot. This would couple the code which retrieves the value with the code that sanitizes it. 

```python
_sanitize(_get(data, "app_name", "via {app_name}"))
```

Reply:
I like this, there was something about the way I was doing it that was bothering me